src/cpu/x64/jit_avx512_core_bf16_conv_kernel.hpp

```diff
@@ -461,8 +467,7 @@ struct jit_avx512_core_bf16_bwd_data_kernel {
     const Xbyak::uint8 *jit_ker() const { return kernel_->jit_ker(); }
 
 private:
-    DNNL_DISALLOW_COPY_AND_ASSIGN(jit_avx512_core_bf16_bwd_data_kernel);
-    jit_generator *kernel_;
+    std::unique_ptr<jit_generator> kernel_;
```

Comment:
Please restore deletion of copy and assignment constructors.

Reply:
okay;  should jit_avx512_core_x8s8s32x_fwd_kernel also then have the macro added? if it’s preferred to have the macros there for documentation, that one was missing the macro.

Reply:
Yes it seems it was missing there.

Reply:
we've updated the bot behavior and regenerated the PR to restore (and add when needed) the macro.

Reply:
FWIW, the copy ctor and assignment operator are deleted for `std::unique_ptr` and therefore they get implicitly deleted for `jit_avx512_core_bf16_bwd_data_kernel `. But I guess it's better to explicitly delete them to show the intent and for the case if we replace `std::unique_ptr` with something else in the future.

Reply:
Our default was to remove the explicit macro for this reason, but our main goal is configuring the bot for consistency and project preferences in cases where the 'right' way is ambiguous.

Reply:
That'd be perfect!

Reply:
I think @mgouicem has not had a chance to re-review yet, but to confirm, are y'all happy with the macros being in explicitly? we've got another one ready to go on the aarch64 side, and i can ship it with or without them!